packages/website/src/css/custom.css

```diff
@@ -211,3 +211,7 @@ h6 {
 td > p:last-child {
   margin-bottom: 0;
 }
+
+h5 {
+  font-weight: bold;
+}
```

Comment:
It was bugging me that the h5s under `ProjectServiceOptions` were roughly the same font size & weight as the text around them...